### PR TITLE
continued following your django starter from github. I think I have e…

### PR DESCRIPTION
**Pull request by hunnypotpy**
…verything here. I cannot get the page to display locally in my browser, I get a : TemplateDoesNotExist at /

base.html

Request Method: 	GET
Request URL: 	http://127.0.0.1:8000/
Django Version: 	3.2.7
Exception Type: 	TemplateDoesNotExist
Exception Value:

base.html

Exception Location: 	/home/potato/Desktop/Git/DjangoBlog/.venv/lib/python3.6/site-packages/django/template/backends/django.py, line 84, in reraise
Python Executable: 	/home/potato/Desktop/Git/DjangoBlog/.venv/bin/python
Python Version: 	3.6.9
Python Path:

['/home/potato/Desktop/Git/DjangoBlog',
 '/usr/lib/python36.zip',
 '/usr/lib/python3.6',
 '/usr/lib/python3.6/lib-dynload',
 '/home/potato/Desktop/Git/DjangoBlog/.venv/lib/python3.6/site-packages']

Server time: 	Tue, 14 Sep 2021 00:15:06 +0000"

Error, hmmm!

### CODE REVIEW

**Comment by bbelderbos** [comment]
Lemme check, btw you probably want to upgrade Python to 3.9?

**Comment by bbelderbos** [comment]
User == `potato` :)

File: mysite/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{% loadstatic %}
```
Comment (bbelderbos) [inline mysite/templates/base.html:1]: typo: `{% load static %}`

**Review by bbelderbos — CHANGES_REQUESTED** [review]

**Comment by bbelderbos** [comment]
You need to add the project wide templates directory (if it was an app one `APP_DIRS` would have located it, like it does for `blog` app templates)

In `mysite/settings.py` update `DIRS`:

```
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [os.path.join(BASE_DIR, 'mysite/templates')],
...
```

Then that error goes away, but you also need to fix `loadstatic` as per review above, then it all renders again.

HTH and good job on the views. Do you want to implement the Django `allauth` plugin next to have users signup and login? If so check out this training:
https://pybites.mykajabi.com/products/pybites-developer-mindset/categories/2567732/posts/10874758

HTH
Bob

File: mysite/templates/base.html (hunk 1 above)
Comment (hunnypotpy) [reply mysite/templates/base.html:1]: I had a feeling it was a typo!!! :)

**Pull request merged**